scripts/calendar/germany_school_holidays_generator.py

```
@@ -0,0 +1,446 @@
+                            (
+                                start_date.year - year,
+                                start_date.month,
+                                start_date.day,
+                                end_date.year - year,
+                                end_date.month,
+                                end_date.day,
+                                holiday_name,
+                            )
+                        )
+
+    for year, year_data in data.items():
```

Ruff (0.15.9)

[warning] Loop control variable `year` not used within loop body

(B007)